src/blocks/rofication.rs

```diff
@@ -155,6 +155,8 @@ fn rofication_status(socket_path: &str) -> Result<RotificationStatus> {
         .read_to_string(&mut buffer)
         .block_error("rofication", "Failed to read from socket")?;
 
+    buffer = buffer.replace("\n", ","); // Original rofication uses NL to separate, while regolith forks uses comma. Convert if needed.
+
     let values = buffer.split(',').collect::<Vec<&str>>();
```

Comment:
You can write `let values = buffer.split_once(|c| c == ',' || c == '\n');` to avoid these two allocations.